app/models/bot/alegre.rb

```diff
@@ -155,10 +155,8 @@ def self.run(body)
         if ['audio', 'image', 'video'].include?(self.get_pm_type(pm))
           self.relate_project_media_async(pm)
         else
-          Bot::Alegre.send_to_media_similarity_index(pm)
-          Bot::Alegre.send_field_to_similarity_index(pm, 'original_title')
-          Bot::Alegre.send_field_to_similarity_index(pm, 'original_description')
-          Bot::Alegre.relate_project_media_to_similar_items(pm)
```

Comment:
Question: Why was this one deleted? Did it become redundant? Is another method doing it?

Reply:
`original_title` and `original_description` definitely get handled by the new logic - the one that doesn't specify.... anything... I am not sure why that was a thing ever in the first place? And it's a media call on a text item? It's maybe just.... wrongly placed? Oh wait! Ok yes - this is because back in the day we'd just hit all four of these for *any* item - and if it was a media request on a text, or a text request on a media, we'd just skip. I'm fairly certain this deletion is the proper move here!

Reply:
So, previously, we were sending the fields to be indexed and then looking for similar items, but now we do both things in one go with `relate_project_media_async`?

Reply:
@caiosba yes mostly - with the caveat that `relate_project_media_async` also gets an explicit call for the fields when indexing/relating text.

Reply:
That is to say - the structure in this version should be the right one- if a non-text, call `relate_project_media_async` with no explicit field, else call with `original_title` and `original_description`